src/lib/index.ts

```diff
@@ -1,4 +1,4 @@
-import { readFile, writeFile } from 'fs-extra';
+import { promises as fs } from 'fs';
```

Comment:
Any reason you're importing `import { promises as fs} from 'fs';` here?

My suggestion would be to rename the import here `import * as fs from 'fs';` and use `fs.promises.writeFile(...)`.  

Alternatively, you could make the import alias to be something like `import { promises as fsPromises } from 'fs';` 🤔

Reply:
@ruheni Thank you for the feedback I would rather use
` import { promises as fsPromises } from 'fs';`  
instead of 
`  import * as fs from 'fs';
`  
BTW the event today was awesome 👍🏻